templates/distribution/manifests/dr/kustomization.yaml.tpl

```diff
@@ -32,6 +37,21 @@ resources:
   - resources/eks-velero-volumesnapshotlocation.yml
 {{- end }}
 
+{{- if eq .spec.distribution.common.provider.type "none" }}
```

Comment:
IMPORTANT: notice that this is also true for the `KFDDistribution` kind.

I guess we can let the user enable etcdBackup with the KFDDistribution kind, having in mind that the (not managed by furyctl) underlying cluster must be made with kubeadm. If we want to do this, **you will need to update the schema for KFDDistribution to include the relevant fields too**.

There are several places where this if condition is repeated.